### ISSUE TITLE
Encourage review comments over issue comments?

### ISSUE THREAD

**Issue by fumieval** [ghc-proposals/ghc-proposals#497]
Top-level issue comments can't be resolved, so derailed conversations tend to fill the page rapidly without being structured.
It would be easier to read conversations if they were hanging at the Unresolved Questions section. Thoughts?

**Comment by nomeata** [comment]
I mostly agree with the goal (with the caveat that resolved discussion are hard to read when you are catching up - but so are very long lists of comments). How would you go about encouraging it, besides maybe leading by example?

**Comment by fumieval** [comment]
Ideally I want issue comments to be exclusive to the review process. How about asking the participants (in a PR description) to avoid discussion about the contents using pull request comments?

**Comment by treeowl** [comment]
I don't understand any of this. What's the problem? What's the proposed solution? It almost sounds like you want to discourage discussion in comments on the pull request, which would seem to make it useless.

**Comment by fumieval** [comment]
> It almost sounds like you want to discourage discussion in comments on the pull request

Yes, that's the point. I want to encourage discussion in __review comments__ so that it's clear which part of the proposal the discussion refers to.

**Comment by treeowl** [comment]
I see... The problem is that only encourages comments on specifics, not discussions of important generalities. I think encouraging review comments is fine, but discouraging others is bad.
